ESSArch_PP/frontend/static/frontend/views/confirm_receive_modal.html

```diff
@@ -0,0 +1,10 @@
+<div class="modal-heading">
+    <h3 class="modal-title" id="modal-title">{{'RECEIVE' | translate}} SIP</h3>
+</div>
+<div class="modal-body" id="modal-body">
+    {{'RECEIVE_SIP_DESC' | translate:$ctrl.data.ip}}
+</div>
+<div class="modal-footer">
+    <button class="btn btn-primary" type="button" ng-click="$ctrl.receive($ctrl.data.ip)" focused="true" ng-disabled="$ctrl.receiving">{{'OK' | translate}}&nbsp;<i ng-if="$ctrl.receiving" class="fas fa-spinner fa-spin"></i></button>
```

Comment:
Move the nbsp inside the ng-if to keep the text centered before clicking the button